src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private UserService use;
+
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = use.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        return password.equals(user.getPassword());
```

Comment:
merge conditions, e.g. could be rewrite in one line
```suggestion
        if (user == null) {
            return false;
        }
        return password.equals(user.getPassword());
```